### ISSUE TITLE
Error loading with `BackboneEncoderUsingLastLayerIntoWPlus`

### ISSUE THREAD

**Issue by anjith2006** [eladrich/pixel2style2pixel#197]
When I set the encoder type as BackboneEncoderUsingLastLayerIntoWPlus, it throws the following error. 
Is the model file different for this one ?
RuntimeError: Error(s) in loading state_dict for BackboneEncoderUsingLastLayerIntoWPlus:
    Missing key(s) in state_dict: "output_layer_2.0.weight", "output_layer_2.0.bias", "output_layer_2.0.running_mean", "output_layer_2.0.running_var", "output_layer_2.3.weight", "output_layer_2.3.bias", "linear.weight", "linear.bias".

**Comment by yuval-alaluf** [comment]
Are you trying to load one of the pre-trained models we provided in the repo? If so, this explains the error. All the final models we uploaded use the `GradualStyleEncoder` which has a different architecture than the `BackboneEncoderUsingLastLayerIntoWPlus`. We provided the `BackboneEncoderUsingLastLayerIntoWPlus` encoder for those who wanted to explore using a lighter weight encoder (which we should in the paper as an ablation study).
